src/DDTrace/Integrations/Mongo/MongoClientIntegration.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
```

Comment:
Given that this is a legacy driver, then I would move it upfront to a Vx folder because we know we will have the new one.

Reply:
Since this is a PECL extension, I based the folder names off of the name of the extension rather than the version number since the version numbers would collide between different extensions. The legacy extension is called [mongo](http://php.net/mongo) and the new one is called [mongodb](http://php.net/mongodb). So I figured we could just put the new extension under the `src/DDTrace/Integrations/MongoDB` folder. Would that work for you? :)

Reply:
Perfect

Reply:
🔝